source/PlayerInfo.cpp

```diff
@@ -210,7 +210,7 @@ void PlayerInfo::Load(const string &path)
 	map<string, map<string, int>> missionCargoToDistribute;
 	map<string, map<string, int>> missionPassengersToDistribute;
 
-	filePath = path;
+	filePath = path.string();
```

Comment:
why is filePath not a path?

Reply:
I don't believe that you can use `.find('~')` on a `path`.

Reply:
I didn't want to refactor more than what's necessary here. Rewriting save handling can be done in a follow-up.